spec/models/namespace_spec.rb

```diff
@@ -7,10 +7,9 @@
 #  created_at  :datetime         not null
 #  updated_at  :datetime         not null
 #  team_id     :integer
-#  registry_id :integer          not null
+#  public      :boolean          default("0")
+#  registry_id :integer
 #  global      :boolean          default("0")
-#  description :text(65535)
-#  visibility  :integer
 #
 # Indexes
 #
```

Comment:
I don't know how others a handling this but the comments are automatically updated by the annotate-models gem.  I can make a new PR and update all file so they are in sync with the gem default.